### ISSUE TITLE
All cameras backing up successfully except for "G3 Instant"

### ISSUE THREAD

**Issue by smshapiro85** [ep1cman/unifi-protect-backup#139]
* Unifi Protect Backup version:
* Unifi Protect version:
* Python version:
* Operating System:
* Are you using a docker container or native?:

### Description

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by ep1cman** [comment]
You've given me absolutely no information to go off of to help resolve this issue.

Please fill in the issue template as well as providing a log of the tool running (with `-vv` passed as an argument to provide more verbose logging)

If I had to take a wild guess with no information, I would check that the unifi user you setup for UPB to login with actually has permissions to access that camera

**Comment by smshapiro85** [comment]
Hi there, I’m so sorry I thought I deleted the post, I didn’t realize I submitted it.

I figured it out mid way through writing the ticket. It turns out the G3 instant does not have person or object detection, so because I was limiting my events to “Person, Vehicle or Ring” the motion events from the G3 instant were not being included.

Therefore, I spun up another container for the g3 instants only, that only includes Motion events, so that I don’t need to include motion on the other 8 cameras that I have which include smart detections.

The app has been working so well, thank you for sharing this solution with the community!
